### DISCUSSION TITLE
Sharing Your BirdNET Pi - Assign Different Ports?

### DISCUSSION THREAD

**Discussion by Rochambeaux** [mcguirepr89/BirdNET-Pi#1121] [Q&A]
Is there a way to change the incoming ports from 443 and 80?  I have another application that I need to reserve those for.

**Comment by pddpauw** [comment]
yes, go to your Caddyfile in /etc/caddy/.  If you add the port to your address, it will use that one!

**Reply by Svardsten53** [reply]
It should work, but then you have to include port numbers in the external URL and define more ports in your router. Another solution is a reverse proxy in your own network. Then you can point different URLs to different IP addresses even if they use the same port. I have done this at home and it works very well.

**Reply by Rochambeaux** [reply]
Is there a way to do this on a Windows box?

**Reply by Svardsten53** [reply]
I have no experience with this in Windows. I use a Synology NAS drive where it works fine. It should also be possible to set up reverse proxy in Caddy (which you already have in the Pi you run BirdNET-Pi in), but then you have to google for the solution.
